modules/oas-page-builder/src/services/pageBuilder.ts

```diff
@@ -274,7 +284,7 @@ export const buildOpenAPIPages = async (
     if (totalSuccess && sourceType == 'atlas') {
       try {
         const gitHash = await fetchGitHash();
-        const versions = await fetchVersionData(gitHash);
+        const versions = await fetchVersionData(gitHash, OAS_FILE_SERVER);
         await saveSuccessfulBuildVersionData(source, gitHash, versions);
```

Comment:
`saveSuccessfulBuildVersionData` currently saves everything to either the `dotcomprd` db or the `dotcomstg` ([mapping code](https://github.com/mongodb/docs-worker-pool/blob/37ef215c1c088a0558d9fa97fdde8907856782a3/modules/oas-page-builder/src/services/database.ts#L30)) since we used to always pull in the latest/production-ready spec. If we're fetching non-prod specs, I think we should either save to the non-prod db. Alternatively, we could just not save at all for non-prod builds (might not be ideal if their dev endpoint also suffers from consistency issues like their prod endpoint once did).

(Tangentially, the old mapping code can probably be deprecated and simplified in favor of using `process.env.SNOOTY_DB_NAME`, similar to how the [persistence module currently does it](https://github.com/mongodb/docs-worker-pool/blob/37ef215c1c088a0558d9fa97fdde8907856782a3/modules/persistence/src/services/connector/index.ts#L31), but that's not necessarily important for this PR)

Reply:
I updated the mapping just so we can save to the non-prod db, let me know if that's what you had in mind